internal/lang/evalchecks/eval_for_each_test.go

```diff
@@ -496,3 +252,752 @@ func TestForEachCommandLineExcludeSuggestion(t *testing.T) {
 		})
 	}
 }
+
+type expectedErr struct {
+	Summary           string
+	Detail            string
+	CausedByUnknown   bool
+	CausedBySensitive bool
+}
+
+func assertDiagnosticsMatch(t *testing.T, diags []tfdiags.Diagnostic, testDiags []expectedErr, phase string) {
+	t.Helper()
+	if len(diags) != len(testDiags) {
+		t.Fatalf("got %d errors in %s phase; expected %d", len(diags), phase, len(testDiags))
+	}
+
+	for i := range diags {
+		assertDiagnosticMatch(t, diags[i], testDiags[i])
+	}
+}
+
+func assertDiagnosticMatch(t *testing.T, diag tfdiags.Diagnostic, testDiags expectedErr) {
```

Comment:
`nitpick`
```suggestion
func assertDiagnosticMatch(t *testing.T, wantDiag tfdiags.Diagnostic, gotDiag expectedErr) {
```

Just for the ease of reading

Reply:
I agree with you that's a good idea, thanks!